server/src/main/java/org/eclipse/lsp/cobol/core/model/tree/variables/VariableDefinitionNode.java

```diff
@@ -39,25 +40,29 @@
  */
 @Getter
 @ToString(callSuper = true)
+@EqualsAndHashCode(callSuper = true)
 public final class VariableDefinitionNode extends Node {
   private int level;
   private VariableNameAndLocality variableName;
   private boolean global;
   private List<String> picClauses;
   private List<OccursClause> occursClauses;
   private List<ValueClause> valueClauses;
-  private List<UsageFormat> usageClauses;
+  @Setter private List<UsageFormat> usageClauses;
```

Comment:
Why do you need a setter for it?

Reply:
To set the usage from parent Node, in case of absence.